rules/windows/privilege_escalation_group_policy_scheduled_task.toml

```diff
@@ -0,0 +1,115 @@
+updated_date = "2021/11/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the modification of Group Policy Object Attributes to execute a scheduled task in the objects controlled by the GPO.
+"""
+index = ["winlogbeat-*", "logs-system.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Scheduled Task Execution at Scale via GPO"
+note = """## Triage and analysis
+
+### Investigating Scheduled Task Execution at Scale via GPO
+
+Group Policy Objects can be used by attackers to execute Scheduled Tasks at scale to compromise Objects controlled by a given GPO,
+this is done by changing the contents of the `<GPOPath>\\Machine\\Preferences\\ScheduledTasks\\ScheduledTasks.xml` file.
+
+#### Possible investigation steps:
+- This attack abuses a legitimate mechanism of the Active Directory, so it is important to determine whether the activity is legitimate
+and the administrator is authorized to perform this operation.
+- Retrive the contents of the `ScheduledTasks.xml` file, check the `<Command>` and `<Arguments>` XML tags for any potentially malicious
+commands and binaries.
+- If the action is suspicious for the user, check for any other activities done by the user in the last 48 hours.
+
+### False Positive Analysis
+- Verify if the execution is allowed and done under change management, and if the execution is legitimate.
+
+### Related Rules
+- Group Policy Abuse for Privilege Addition
+
+### Response and Remediation
+- Immediate response should be taken to validate activity, investigate and potentially isolate activity to prevent further
+post-compromise behavior.
+
+## Config
+
+The 'Audit Detailed File Share' audit policy is required be configured (Success Failure).
+Steps to implement the logging policy with with Advanced Audit Configuration:
+```
+Computer Configuration > 
+Policies > 
+Windows Settings > 
+Security Settings > 
+Advanced Audit Policies Configuration > 
+Audit Policies > 
+Object Access > 
+Audit Detailed File Share (Success,Failure)
+```
+
+The 'Audit Directory Service Changes' audit policy is required be configured (Success Failure).
+Steps to implement the logging policy with with Advanced Audit Configuration:
+```
+Computer Configuration > 
+Policies > 
+Windows Settings > 
+Security Settings > 
+Advanced Audit Policies Configuration > 
+Audit Policies > 
+DS Access > 
+Audit Directory Service Changes (Success,Failure)
+```
+"""
+references = [
+    "https://github.com/atc-project/atc-data/blob/master/docs/Logging_Policies/LP_0025_windows_audit_directory_service_changes.md",
+    "https://github.com/atc-project/atc-data/blob/f2bbb51ecf68e2c9f488e3c70dcdd3df51d2a46b/docs/Logging_Policies/LP_0029_windows_audit_detailed_file_share.md",
+    "https://labs.f-secure.com/tools/sharpgpoabuse",
+    "https://twitter.com/menasec1/status/1106899890377052160",
+    "https://github.com/SigmaHQ/sigma/blob/master/rules/windows/builtin/win_gpo_scheduledtasks.yml"
+]
+risk_score = 73
```

Comment:
maybe risk_score should be set to Medium (cuz still need to check the task content/action before marking it as high/80% suspicious).

Reply:
Done